addon/components/power-calendar/days.js

```diff
@@ -18,6 +18,16 @@ function withLocale(locale, fn) {
   return returnValue;
 }
 
+const WEEK_DAYS = [
```

Comment:
I also find weird creating an array from a string when you can create the array directly.